packages/core/src/domain/tracekit/computeStackTrace.spec.ts

```diff
@@ -499,6 +499,53 @@ Error: foo
     })
   })
 
+  it('should ignore custom error messages on chrome ', () => {
+    const stackFrames = computeStackTrace(new Error('bar@http://path/to/file.js:1:1'))
+
+    expect(stackFrames.stack[0]).not.toEqual({
+      args: [],
+      column: 1,
+      func: 'Error: bar',
+      line: 1,
+      url: 'http://path/to/file.js',
+    })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** If anything change in the StackFrame format, this test will succeed without actually testing the expectation. Maybe only test for the URL?

```suggestion
    expect(stackFrames.stack[0].url).not.toBe('http://path/to/file.js')
```